### ISSUE TITLE
Authentication Failure Despite Correct Credentials on `get_datasets()`

### ISSUE THREAD

**Issue by truenomad** [ropensci/rdhs#157]
Hi there!

Thank you for this great package!

I am encountering an authentication error when using the `get_datasets()` function from the rdhs package. Despite providing the correct credentials and being able to download the data manually from the DHS website without any issues, the function fails to authenticate and returns an error message. The code to reproduce this issue looks this:

```
# Set configuration
rdhs::set_rdhs_config(
  email = "myemail.int",
  project = "Country analytics",
  config_path = here::here("rdhs.json"), 
  data_frame = "data.table::as.data.table",
  password_prompt = TRUE,
  global = FALSE
)

# Get the latest PR surveys
datasets_PR_SV <- rdhs::dhs_datasets(
  fileFormat = "SV", 
  fileType = "PR"
) |> 
  dplyr::group_by(CountryName) |> 
  dplyr::slice_max(SurveyYear) 

# Attempt to download the first dataset
downloads <- rdhs::get_datasets(datasets_PR_SV$FileName[1])
```

Error message I get when running the above:

```
Logging into DHS website...
Error in authenticate_dhs(config) : 
  Your log in credentials were not recognised by the DHS website.
Please check your credentials are right (?get_rdhs_config), and your internet connection for possible error.
```
Some additional info on the issue and the things I have done:

1) I have verified that my credentials are correct by logging into the DHS website and manually downloading the data without any issues.
2) Internet connectivity is stable, and other network operations in R are functioning correctly.
3) I have the necessary permissions and project approval to access the datasets.
4) The password is entered via the prompt after running `set_rdhs_config`.

Any assistance in resolving this authentication issue would be greatly appreciated. Many thanks once again!

**Comment by jeffeaton** [comment]
Thanks @truenomad. This may be a question best suited for @OJWatson.

But just to confirm, to check that `set_rdhs_config()` has saved the information correctly, when you call `get_rdhs_config()` does it return the correct email and password? 

```
> get_rdhs_config()
$email
[1] "jeffrey.eaton@imperial.ac.uk"

$project
[1] "Project to estimate trends"

$password
[1] "abcdefgh123"
...
```

**Comment by truenomad** [comment]
Thanks, @jeffeaton. 

Yes, the 'set_rdhs_config' worked well and I'm able to see my credentials when I call 'get_rdhs_config'.

**Comment by truenomad** [comment]
I have managed to solve the above issue. The project name that I declared in `set_rdhs_config` and the one I had on the DHS website did not match. Once I updated the project argument to match the exact project name, the `get_datasets()` function worked perfectly. 

It might be helpful to include a specific error message or note in the documentation to alert users to ensure that the project name in their code matches exactly with the one on the DHS website.
